public/index.html

```diff
@@ -24,6 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+
+    <!-- Fonts -->
```

Comment:
Please add only the font weight that we use.

Reply:
Done.